tools/postinstall/sass_worker_async.patch

```diff
@@ -1,15 +1,21 @@
 diff --git sass/sass_wrapper.js sass/sass_wrapper.js
-index 21abb8f..9c750a3 100644
+index ad1f7a6..957d56e 100644
 --- sass/sass_wrapper.js
 +++ sass/sass_wrapper.js
-@@ -17,7 +17,11 @@ const fs = require('fs');
+@@ -17,7 +17,17 @@ const fs = require('fs');
  const args = process.argv.slice(2);
  if (runAsWorker(args)) {
    debug('Starting Sass compiler persistent worker...');
--  runWorkerLoop(args => sass.run_(args));
+-  runWorkerLoop(args => sass.cli_pkg_main_0_(args));
 +  runWorkerLoop(args => {
 +    return new Promise(resolve => {
-+      sass.run_(args)['then$1$2$onError'](() => resolve(true), () => resolve(false));
++      const result = sass.cli_pkg_main_0_(args);
++      const originalThen = result.then$1$2$onError;
++      let dynamicType;
++      result.then$1$2$onError = (receiver, done, error, type) => dynamicType = type;
```

Comment:
This is a very janky implementation, but the problem it's trying to solve is that `then$1$2$onError` requires an internal Dart type object to be passed in which we don't have access to. I hacked around it by monkey-patching the function, saving a reference and then passing it back in.